pkg/manager/task_clean.go

```diff
@@ -60,6 +56,11 @@ func (t *CleanTask) shouldCleanScene(s *models.Scene) bool {
 		return true
 	}
 
+	if matchFile(s.Path, config.GetExcludes()) {
```

Comment:
Imo the matchFiles for the scenes,galleries,images should be checked after the checks for Exclude.Video, Exclude.Image, extension.